doc/influxql-2.md

```diff
@@ -0,0 +1,910 @@
+
+#### Time
+
+The time block can be specified as either an epoch or an RFC3339Nano string. If an epoch is used it should be followed by its precision. However, if no precision is specified, the system will guess the precision by whatever is closest to the current time on the server receiving the write.
+
+#### Meta data
+
+*TODO:* give this more definition.
+
+Meta data can be written with a line starting with `##`. The meta data will be associated with the series on the next line. Meta data is a JSON object. Keys in the object starting with `_` are reserved for the system. The data in meta is not indexed. The only validation is that it is a parseable JSON object.
+
+#### Examples
+
+```text
+system:cpu,region:cpu,host:a,metric:user_idle 23.2 1491675816
+system:cpu,region:cpu,host:a,metric:user_idle 23.2 1491675816s
+system:cpu,region:cpu,host:a,metric:user_idle 23.2 1491675816002ms
+system:cpu,region:cpu,host:a,metric:user_idle 23.2 2017-04-08T14:23:54Z
+etype:error,app:paul's\ app t 1491675816000000
+some\:key:foo 2i 1491675816
+service:apache,host:serverA,region:west "127.0.0.1 - frank [10/Oct/2000:13:55:36 -0700] \"GET /apache_pb.gif HTTP/1.0\" 200 2326" 2017-04-18T14:58:00Z
+```
+
+#### gRPC
+
+Writes can come in through gRPC. Here's the `proto` definition for the schema and service. (TODO)
+
+### Representing the InfluxDB 1.x Data Model
+
+The InfluxDB 1.x data model can be represented by using special `_measurement` and `_field` tag keys to split measurement and fields up. Here's an example:
+
+```text
+# InfluxDB 1.x
+cpu,host=A,region=west user=16.4,system=23.2 1491675816
+
+# Influx 2.0 example
+_measurement:cpu,host:serverA,region:west,_field:user 16.4 1491675816
+_measurement:cpu,host:serverA,region:west,_field:system 23.2 1491675816
+```
+
+### Representing the Prometheus/OpenTSDB Data Model
+
+The Prometheus data model can be represented by using the special `_metric` tag key. Here's an example:
+
+```text
+# Prometheus
+node_cpu{cpu="cpu0",mode="user"} 182168.46875
+
+# Influx 2.0
+_metric:node_cpu,cpu:cpu0,mode:user 182168.46875 1491675816
+```
+
+### Pre-defined tag keys
+
+There are a set of tag keys used by the system that start with `_` that are well known. Some of them are for conversions from Influx 1.x or Prometheus while others are for information useful in graphing.
+
+* **_units** - the units of the measurement - 
+* **_type** - the type of the series - [int64, uint64, float64, bool, string]
+* **_samplingRate** - duration string for how frequently this series is expected to be sampled. `0` indicates that it is an event stream (irregular series)
```

Comment:
should we call it  **_samplingInterval**? Sampling rate is assumed to be number of samples in **one** second for people with electronics background.